lib/neurx/structures/network.ex

```diff
@@ -34,15 +35,16 @@ defmodule Neurx.Network do
   @doc """
   Return the network by pid.
   """
-  def get(pid), do: Agent.get(pid, & &1)
-
+  def get(pid) do
+    GenServer.call(pid, {:get})
+  end
```

Comment:
Add whitespace between the "end" and the "@doc" to make it consistent with the rest of the file.